docs/content/guides/dagster/graph_job_op.mdx

```diff
@@ -279,44 +275,45 @@ def do_it_all():
     do_something()
 ```
 
-With the new APIs, you can accomplish this by supplying config when building the job.
+With the new APIs, you can accomplish this by supplying config to the job.
 
 ```python file=/guides/dagster/graph_job_op/graph_with_config.py
-from dagster import graph, op
+from dagster import job, op
 
 
 @op(config_schema={"param": str})
 def do_something(_):
     ...
 
 
-@graph
-def do_it_all():
-    do_something()
+default_config = {"ops": {"do_something": {"config": {"param": "some_val"}}}}
 
 
-do_it_all_job = do_it_all.to_job(
-    config={"solids": {"do_something": {"config": {"param": "some_val"}}}}
-)
+@job(config=default_config)
+def do_it_all():
+    do_something()
```

Comment:
re: config weirdness, i think as for "preset" this default feeling (predefine config on def) is a lot better than `.to_job(config)`

Reply:
+1.  I think it would look a little nicer with the config inlined into the job definition instead of as its own variable.